src/mapkit.cs

```diff
@@ -1974,5 +2064,312 @@ interface MKUserTrackingButton {
 		[NullAllowed, Export ("mapView", ArgumentSemantic.Weak)]
 		MKMapView MapView { get; set; }
 	}
+
+	[TV (13,0), NoWatch, Mac (10,15), iOS (13,0)]
+	[BaseType (typeof(NSObject))]
+	interface MKPointOfInterestFilter : NSSecureCoding, NSCopying
+	{
+		[Static]
+		[Export ("filterIncludingAllCategories")]
+		MKPointOfInterestFilter FilterIncludingAllCategories { get; }
+
+		[Static]
+		[Export ("filterExcludingAllCategories")]
+		MKPointOfInterestFilter FilterExcludingAllCategories { get; }
+
+		[Internal]
+		[Export ("initIncludingCategories:")]
+		IntPtr InitIncludingCategories (string[] categories);
+
+		[Internal]
+		[Export ("initExcludingCategories:")]
+		IntPtr InitExcludingCategories (string[] categories);
+
+		[Export ("includesCategory:")]
+		bool IncludesCategory (string category);
+
+		[Export ("excludesCategory:")]
+		bool ExcludesCategory (string category);
+
+		[TV (13, 0), NoWatch, Mac (10, 15), iOS (13, 0)]
+		[Field ("MKPointOfInterestCategoryAirport")]
+		NSString Airport { get; }
```

Comment:
that (and the following) looks like a good case to use a smart enum

and it also points out that `category` (in the API above) should be `NSString` (not `string`) with a `[BindAs]` to the smart enum